src/milind/PrimeNumberAndAdditionAndAverage.java

```diff
@@ -0,0 +1,34 @@
+package milind;
+
+import java.util.Arrays;
+
+public class PrimeNumberAndAdditionAndAverage {
+	
+	void primeNumber(int startRange, int endRange) {
+		int sum = 0;
+		int countPrimeNumber = 0;
+		for(int index = startRange; index <= endRange; index++) {
+			int count = 0;
+			for(int innerIndex = startRange; innerIndex <= index; innerIndex++) {
+				if(index % innerIndex == 0) {
+					count++;
+				}
+			}
+			if(count <= 1) {

```

Comment:
here you should write only < condition
after doing above code change think for 13
13%2
13%3 n so on for no single number if condition will satisfied so cnt will be 0,
better to use boolean flag instead of int counter